### ISSUE TITLE
B+R+I cheat can't be triggered on low-end keyboards due to rollover

### ISSUE THREAD

**Issue by freeidiot40x2** [jorio/OttoMatic#12]
Hello @jorio, one of the first games that I played with my niece and nephew when I was in college. Thank you very much for updating this game. The Windows version works very well (using it in Windows 10 right now). The level cheat code (Pressing space +F10 to go to any level) works fine. However, the full health and power cheats (pressing I+F+D together) does not seem to work. I confirm that this used to work in the original Windows port bought out by Ideas from the deep (have a copy which I purchased long ago and installed it to checks this out). Would be great if this is fixed:).

**Comment by jorio** [comment]
Thank you for your feedback! The cheat you're looking for is triggered by pressing B+R+I together, as in the original Mac release of the game. Other cheats are described in [CHEATS.md](https://github.com/jorio/OttoMatic/blob/master/CHEATS.md).

**Comment by freeidiot40x2** [comment]
Hello @jorio: I do not understand. Pressing B+R+I does not do anything in the Windows version. However, the other cheats appear to be working fine.

**Comment by jorio** [comment]
So I gave it a try again and I think I know what's happening.

I confirm that the Windows version of the game does support B+R+I. However, some keyboards have hardware limitations on which keys can be pressed simultaneously. I just tried a cheapo USB keyboard that I had lying around and sure enough B+R+I wouldn't register, whereas I'd never had that problem with my regular keyboard.

That's probably why IFD changed the cheat to I+F+D for their Windows port, as these keys are more likely to work together on most keyboards.
